mne/viz/_figure.py

```diff
@@ -447,6 +447,13 @@ def __init__(self, inst, figsize, ica=None, xlabel='Time (s)', **kwargs):
             # hide some ticks
             ax_main.tick_params(axis='x', which='major', bottom=False)
             ax_hscroll.tick_params(axis='x', which='both', bottom=False)
+        elif self.mne.show_real_time:
+            if not isinstance(self.mne.show_real_time, str):
+                self.mne.show_real_time = "%H:%M:%S"
+            for _ax in (ax_main, ax_hscroll):
+                _ax.xaxis.set_major_formatter(
+                    FuncFormatter(self._xtick_timestamp_formatter))
+                _ax.set_xlabel(f'Time ({self.mne.show_real_time})')
```

Comment:
```suggestion
                _ax.set_xlabel(f'Time (hour/min/sec)')
```
less geeky

Reply:
Or maybe `Time (hh:mm:ss)` (should also be less geeky but more exact)...

Reply:
This was supposed to change, when the user provides its own datetime-format.

Originally I thought it would be nice, if you can decide the datetime-format yourself by providing the datetime format code.
But probably there is not that much variation from the default format H/M/S, which would be more useful for the user, right?
**Should we just provide the default?**
That would also make implementing the zoom-in/zoom-out format-changes more straight-forward?

Reply:
Letting users pass a custom datetime format code seems like overkill to me. HH:MM:SS.SSS ought to be adequate for all users needs. If it's not, we can add that enhancement only when someone requests it.